src/shared/components/driverAnnotations/DriverAnnotationControls.tsx

```diff
@@ -244,69 +245,74 @@ export default class DriverAnnotationControls extends React.Component<
                                         </label>
                                     </div>
                                 )}
-                            {this.props.handlers
-                                .onChangeAnnotateCBioPortalInputValue && (
-                                <div className="checkbox">
-                                    <label>
-                                        <input
-                                            type="checkbox"
-                                            value={EVENT_KEY.annotateCBioPortal}
-                                            checked={
+                            {getServerConfig().show_oncokb_cbiocosmic &&
```

Comment:
Update property names after update of backend branch.

Reply:
fixed